src/Simplic.FileStructure.Workflow.Data.DB/DocumentWorkflowTrackerRepository.cs

```diff
@@ -26,7 +26,7 @@ public bool IsDocumentUserAssigned(Guid documentId, int userId)
             return sqlService.OpenConnection((connection) =>
             {
                 return connection.Query<bool>($"Select case when Exists(Select * from {TableName} where DocumentId =:documentId " +
-                    $"and UserId = :userId) Then '1' Else '0' End as Result ",
+                    $"and UserId = :userId) Then 'True' Else 'False' End as Result ",
```

Comment:
```suggestion
                    $"and UserId = :userId) Then 1 Else 0 End as Result ",
```

Looks more reaonable and has less dependency to the db driver. What do you think?